elastalert/kibana.py

```diff
@@ -166,6 +166,8 @@
                   u'style': u'dark',
                   u'title': u'ElastAlert Alert Dashboard'}
 
+kibana4_time_temp = "(refreshInterval:(display:Off,section:0,value:0),time:(from:'%s',mode:absolute,to:'%s'))"
```

Comment:
I also have a style concern here: please include spaces after commas for legibility and PEP8.